### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by rachelcarmena**

### CODE REVIEW

File: docs/_code/optics-home-code.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ fun main() {
 
 
   val john =
-  Employee("John Doe",
+  Employee("Audrey Tang",
```
Comment (aballano) [inline docs/_code/optics-home-code.md:40]: ❤️

File: docs/_data/doc-versions.yml [hunk 2]
```diff
@@ -5,10 +5,14 @@ docVersions:
       previous: https://arrow-kt.io/docs/0.8/
 
     - title: "v0.10.5"
-      url: https://arrow-kt.io/
+      url: https://arrow-kt.io/docs/0.10/
```
Comment (aballano) [inline docs/_data/doc-versions.yml:8]: shouldn't be 

```suggestion
      url: https://arrow-kt.io/docs/0.10.5/
```

?

File: docs/_data/doc-versions.yml [hunk 3]
```diff
@@ -5,10 +5,14 @@ docVersions:
...
       previous: https://arrow-kt.io/docs/0.9/
 
-    - title: "v0.11.0-SNAPSHOT"
+    - title: "v0.11.0"
+      url: https://arrow-kt.io/
+      previous: https://arrow-kt.io/docs/0.10/
```
Comment (aballano) [inline docs/_data/doc-versions.yml:13]: if so, same here :)

**Review by aballano — APPROVED** [review]
good catch on that either doc fix 👌

File: docs/_data/doc-versions.yml (hunk 2 above)
Comment (rachelcarmena) [reply docs/_data/doc-versions.yml:8]: Thanks for the review @aballano :raised_hands: Just `major.minor` for the website.

**Pull request merged**